docs/research/user-experience/xmr-btc-user-interface-validation.md

```diff
@@ -0,0 +1,48 @@
+# XMR<>BTC Atomic Swap - User Interface Prototype Validation
+
+This document sums up the questions that we would like to validated with mocks.
+
+## Questions
+
+The questions are split between maker (liquidity provider) and taker (normal user), because the objectives are somewhat different.
+
+Is a `taker` happy to...
```

Comment:
I think the points cover pretty much what we want to find answers to. 

A thought when reading them: 
At the end it boils down to how you ask the user the question. The answer to `Are you happy running a bitcoind full-node` will be different to `Would you trust a third party with monitoring the blockchain for you`. 

It might help us to change our own mindset if we phrase these points differently in from of questions to ourselves e.g. 

* `How does a user/taker/maker currently broadcast/monitor bitcoin transactions?`
  * `Is a user fine with trusting a third party such as a block explore (blockchain.com/blockstream.com)?` 
* `How does a user/taker/maker hide its identity (if at all)?`

If you want, I can help you rephrasing some of the points.

Reply:
> I think the points cover pretty much what we want to find answers to.
> 
> A thought when reading them:
> At the end it boils down to how you ask the user the question. The answer to `Are you happy running a bitcoind full-node` will be different to `Would you trust a third party with monitoring the blockchain for you`.

Just so we are clear, we are not asking a user directly.
At this stage, we assume usage of full node. If we were to move into a direction of not requiring full node, most likely because the user uses another wallet, then yes indeed we would have to find a way to show the user that we are sending data to an blockchain explorer. Maybe via a note or maybe letting the user configure which explorer.

> It might help us to change our own mindset if we phrase these points differently in from of questions to ourselves e.g.
> 
>     * `How does a user/taker/maker currently broadcast/monitor bitcoin transactions?`

I agree, we should add a question `What bitcoin/monero wallet does the user want to use to swap?`

@da-kami maybe the "is happy" format to restrictive to properly phrase the questions we want to answer.

>       
>       * `Is a user fine with trusting a third party such as a block explore (blockchain.com/blockstream.com)?`

as stated above, I am not sure this is relevant at this stage.

>     * `How does a user/taker/maker hide its identity (if at all)?`

This question is very broad and I believe make it hard to validate in a mock.
I prefer to see `Is the user happy to contact other party using clear ip?` so that we know the answer is yes or no.

> 
> If you want, I can help you rephrasing some of the points.

We are not sure to understand what your point is. These are not interview questions and you seem to think they are by the way you started your reply. Can you please clarify a bit more what issue you see with the current format of the questions?